_utils/terror_gen/errors_release.txt

```diff
@@ -159,13 +159,13 @@ ErrTaskCheckSameTableName,[code=26001:class=task-check:scope=internal:level=medi
 ErrTaskCheckFailedOpenDB,[code=26002:class=task-check:scope=internal:level=high],"failed to open DSN %s:***@%s:%d"
 ErrTaskCheckGenTableRouter,[code=26003:class=task-check:scope=internal:level=medium],"generate table router error"
 ErrTaskCheckGenColumnMapping,[code=26004:class=task-check:scope=internal:level=medium],"generate column mapping error"
-ErrTaskCheckSyncConfigError,[code=26005:class=task-check:scope=internal:level=medium],"%s %v: %v\n detail: %v"
+ErrTaskCheckSyncConfigError,[code=26005:class=task-check:scope=internal:level=medium],"%s: %v\n detail: %v"
 ErrTaskCheckGenBWList,[code=26006:class=task-check:scope=internal:level=medium],"generate black white list error"
 ErrRelayParseUUIDIndex,[code=28001:class=relay-event-lib:scope=internal:level=high],"parse server-uuid.index"
 ErrRelayParseUUIDSuffix,[code=28002:class=relay-event-lib:scope=internal:level=high],"UUID (with suffix) %s not valid"
 ErrRelayUUIDWithSuffixNotFound,[code=28003:class=relay-event-lib:scope=internal:level=high],"no UUID (with suffix) matched %s found in %s, all UUIDs are %v"
 ErrRelayGenFakeRotateEvent,[code=28004:class=relay-event-lib:scope=internal:level=high],"generate fake rotate event"
-ErrRelayNoValidRelaySubDir,[code=28005:class=relay-event-lib:scope=internal:level=high],"no valid relay sub directory exists"
+ErrRelayNoValidRelaySubDir,[code=28005:class=relay-event-lib:scope=internal:level=high],"there is no relay data under relay directory %s. please check relay data and status"
```

Comment:
```suggestion
ErrRelayNoValidRelaySubDir,[code=28005:class=relay-event-lib:scope=internal:level=high],"there aren't any data under relay log directory %s. please check relay log using  query-status"
```

is it better?

Reply:
Yes, addressed in https://github.com/pingcap/dm/pull/510/commits/bae91a62f440e7a362fc2f39ffd091206316726e